gems/aws-crt/ext/compile.rb

```diff
@@ -41,15 +41,16 @@ def find_file(name, search_dirs, base_dir)
 end
 
 # Compile bin to expected location
-def compile_bin
-  platform = local_platform
+def compile_bin(cpu)
```

Comment:
I know we're defaulting cpu in the bin task, but we may want to add a `cpu ||= host_cpu` here to default this as well.